dbc2val/dbcfeederlib/databroker.py

```diff
@@ -53,32 +59,19 @@ def register(self, name: str, data_type: Union[str, DataType], description: str)
             if len(metadata) == 1:
```

Comment:
the log.debug above: Should it say "register", becasue it does not try to register, just checking